contrib/nightly/create_tarball.sh

```diff
@@ -174,46 +174,51 @@ do_command "git clone $giturl ompi"
 cd ompi
 do_command "git checkout $gitbranch"
 
-# Find the "git describe" string for this branch (remove a leading "ompi-"
-# prefix, if there is one).
-describe=`git describe --tags --always | sed -e s/^ompi-//`
+# Nightly tarballs are named in this format:
+# openmpi-${BRANCHNAME}-${YYYYMMDDHHMM}-${SHORTHASH}.tar.${COMPRESSION}
+timestamp=`date '+%Y%m%d%H%M'`
+githash=`git show '--pretty=format:%h'`
```

Comment:
I got curious and wanted to see what git command would show.  For me it ends up being a long string starting with the shortened hash.